src/components/Highlight/index.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from "@emotion/styled";
+import { palette } from "@leafygreen-ui/palette";
+
+const { red } = palette;
+
+const Highlight = styled.mark<{ "data-cy"?: string }>`
+  background-color: ${red.light2};
```

Comment:
nit. this could use some opacity. this is kind of sad but I think we can't use the color tokens with `rgba` so we would need to use `#rrggbbaa` format

here's a list of transparency codes: [link](https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4)

Reply:
This will be handled in 
https://jira.mongodb.org/browse/EVG-18030